### PR TITLE
[stable/sonarqube] Use #!/bin/bash in ./templates/copy-plugins.yaml instead of #!/bin/sh

### LINKED ISSUE #13482: [stable/sonarqube] `extraEnv` not passed to /opt/sonarqube/bin/run.sh when `.Values.plugins.install` is truthy
**Describe the bug**
When `.Values.plugins.install` evaluates as truthy, environment variables set via `.Values.extraEnv` are not available in the SonarQube container.

**Which chart**:
stable/sonarqube

**What happened**:
Setting `extraEnv: sonar.web.context: /sonar` has no effect on the started process.  Examining the running JAR with `jps -lvm` shows that it was not started with the -Dsonar.web.context=/sonar argument.

**What you expected to happen**:
The JAR should have been started with the `-Dsonar.web.context=/sonar` argument.

**How to reproduce it** (as minimally and precisely as possible):
1. Enable plugin installation (any plugin will do)
2. Set a SonarQube environment variable in `extraEnv` (the startup script scans env for properties that begin with `sonar.`)
3. Examine the running process with `jps`; it will not have been started with the specified environment variable as an argument, e.g:

`kubectl exec -it {sonarqube_pod} -- /bin/bash -c "/usr/lib/jvm/java-1.8.0-openjdk-amd64/bin/jps -lvm" | grep sonar-application`

Which should output something like:

`lib/sonar-application-7.7.jar -Dsonar.log.console=true -Dsonar.jdbc.username=sonarUser -Dsonar.jdbc.password=sonarPass -Dsonar.jdbc.url=jdbc:postgresql://full-arachnid-postgresql:5432/sonarDB -Dsonar.web.javaAdditionalOpts= -Djava.security.egd=file:/dev/./urandom`

Note that `-Dsonar.web.context=/sonar` is missing.

**Anything else we need to know**:
I believe this is due to the shebang in `./templates/copy-plugins.yaml`.  The shebang is `#!/bin/sh`; setting it to `#!/bin/bash` or `#!/usr/bin/env bash` fixes the issue.

### PR DESCRIPTION
**Pull request by out-of-band**
#### What this PR does / why we need it:
Small fix to resolve issues caused by using the init container used to install plugins.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #13482

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/sonarqube/templates/copy-plugins.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   copy_plugins.sh: |-
-      #! /bin/sh
+      #!/bin/bash
```
Comment (cpanato) [inline stable/sonarqube/templates/copy-plugins.yaml:12]: why?

Comment (out-of-band) [reply stable/sonarqube/templates/copy-plugins.yaml:12]: I linked the issue it resolves in the PR: https://github.com/helm/charts/issues/13482

**Comment by out-of-band** [comment]
Any feedback on this?

I am unable to both install plugins and use `extraEnv` without this change (meaning I can't use this chart at all in my environment because I have to install SonarQube at domain.com/sonar, not sonar.domain.com, and I need to set the `sonar.web.context` environment variable).

**Comment by rjkernick** [comment]
You just need to merge in the latest master branch and modify the version in chart.yaml once more. Don't forget --signoff when you merge in master.

**Comment by out-of-band** [comment]
Merged and bumped the version and signed off.  Thanks!

**Pull request merged**